files/image_config/topology/topology.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Internal topology service
+Requires=database@0.service database@1.service database@2.service database@3.service database@4.service database@5.service
```

Comment:
why hard code numbers here?

Reply:
Fixed to dynamically update dependency based on number of asics